doc/api/tls.md

```diff
@@ -789,8 +790,6 @@ added: v0.11.3
 * `port` {number}
 * `host` {string}
 * `options` {Object}
-  * `host` {string} Host the client should connect to.
```

Comment:
I noticed this, too, I'll be PRing fixes to the tls.connect docs soon. Btw, options port/host overrides args port/host, which is opposite of what I would expect :-(